starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,98 @@
+* {
+  margin:0;
+  padding:0;
+  box-sizing: border-box;
+}
+
+header {
+  background-image: url("/images/apple-pie.jpg");
```

Comment:
to get your background image to work, you need to do `background-image: url("../images/apple-pie.jpg"); `

This is because you need to go up one level to reach the image folder

Reply:
Oh ok! it worked on my lite-server but I didn't check it directly by opening it in the browser. Thanks!